.babelrc

```diff
@@ -1,8 +1,10 @@
 {
 	"sourceMap": "inline",
-	"presets": ["es2015-loose", "stage-0", "react"],
+	"presets": [
+		["@babel/env", { "targets": { "esmodules": true } }],
```

Comment:
Babel officially deprecated `stage-0`. Though `@babel/preset-env` is less specific here than the old plugin was, I believe a) the fact that this library was written in a very future-forward method, and b) the Babel ecosystem has improved since this was written, I think this functionally works out to the same output. This is just the now-supported way to do it.